tests/ci/README.md

```diff
@@ -0,0 +1,65 @@
+# Tests
+
+This directory contains the configuration, scripts, and files for building an
+image using the "RHEL for Edge" tools, and to test the result.
+
+For running ths part of the CI pipeline, we use [tmt](https://tmt.readthedocs.io/en/stable/overview.html#overview),
```

Comment:
```suggestion
For running this part of the CI pipeline, we use [tmt](https://tmt.readthedocs.io/en/stable/overview.html#overview),
```

Reply:
Thanks I integrated the change with the new changes.